.github/workflows/main.yml

```diff
@@ -43,6 +43,15 @@ jobs:
           echo $GITHUB_WORKSPACE;
           meson test --wrap=$GITHUB_WORKSPACE/valgrind/valgrind.sh --verbose -C builddir
 
+      - name: Test Reference
+        run:
+          export GST_PLUGIN_PATH=${GITHUB_WORKSPACE}/builddir/plugins;
+          echo $GITHUB_WORKSPACE;
+          ./builddir/test/testapp samples/Sample_10.avc > builddir/Sample_10.avc.ref;
+          diff builddir/Sample_10.avc.ref samples/Sample_10.avc.ref;
```

Comment:
would be great to make the CI fail if diff outputs a difference

Reply:
it fails

Reply:
For example this one https://github.com/Igalia/GstVkVideoParser/actions/runs/3584298073